web/src/components/snapshots/SnapshotDetails.jsx

```diff
@@ -980,9 +980,7 @@ class SnapshotDetails extends Component {
           <ShowAllModal
             displayShowAllModal={showAllPostSnapshotScripts}
             toggleShowAllModal={this.toggleShowAllPostScripts}
-            dataToShow={this.renderShowAllPostscripts(
-              this.postSnapshotScripts()
-            )}
+            dataToShow={this.renderShowAllScripts(this.postSnapshotScripts())}
             name="Post-snapshot scripts"
```

Comment:
renderShowAllPostscripts function never existed in the code base. However, these should render similarly to pre-snapshot scripts. replacing it with the same function here.